delete.go

```
@@ -18,7 +18,7 @@ func killContainer(container *libcontainer.Container) error {
 	for i := 0; i < 100; i++ {
 		time.Sleep(100 * time.Millisecond)
 		if err := container.Signal(unix.Signal(0)); err != nil {
-			destroy(container)
+			_ = destroy(container)
```

Why ignore the error here?

---

Just keep others unchanged to make the minimum changes.